pkg/collector/corechecks/oracle-dbm/activity.go

```diff
@@ -357,6 +357,13 @@ func (c *Check) SampleSession() error {
 		if sample.PdbName.Valid {
 			sessionRow.PdbName = sample.PdbName.String
 		}
+		if sessionRow.PdbName == "" {
+			if c.multitenant {
+				sessionRow.PdbName = "CDB$ROOT"
+			} else {
+				sessionRow.PdbName = c.cdbName
```

Comment:
are we at risk of overloading the `PdbName` field? Would it be better to only send strictly correct data from the agent, and resolve `database` to the correct name on the backend?

Reply:
> are we at risk of overloading the `PdbName` field? Would it be better to only send strictly correct data from the agent, and resolve `database` to the correct name on the backend?

In the multitenant architecture, known as CDB (Container Database), the native identifiers `CdbName` and `PdbName` correspond to the root container and pluggable database, respectively. In this setup, both fields need to be populated. On the other hand, in the legacy architecture (a.k.a non-CDB, non-multitenant), the hierarchical structure of CDB->PDB is absent, and only a single database exists. To accommodate the legacy architecture without introducing a new field like `database` and making backend and frontend adjustments, we opted to reuse the existing model. This decision addressed the immediate need for legacy architecture support. A potential remodelling should involve a broader discussion outside of this pull request. Factors to consider:
-  the absence of significant drawbacks thus far, with the exception of the current issue, 
- de-supporting legacy architecture in the latest Oracle releases.